Improve 120m global mosaic description

mlubej opened the pull request:
Changes:
- improved description
- added field for QM band description in the README.md
- whitespace removal
- new animations
- resize images in README.md
- other small fixes

mlubej commented:
@Dorothyrono I can't add you as a reviewer, but here is a PR with some suggested changes for the collection description. 

Is there a way to check how the new yaml renders before it goes online?

Dorothyrono commented:
@mlubej thanks alot  for the improvements,

yes we do  normally check with a  local build by running  `npm run build ` which then creates an `output `directory with html pages of all collections

but first  add .env file to root  with the following contents
```
GIT_HUB_COLLECTIONS_REPO=sentinel-hub/public-collections
GIT_HUB_COLLECTIONS_BRANCH=main
COLLECTIONS_BROWSER_ROOT_URL="https://collections.sentinel-hub.com/"
COLLECTIONS_BROWSER_S3_BUCKET=s3://collections.sentinel-hub.com/

```
could you try this local build  ?

Dorothyrono commented on collections/sentinel-s2-l2a-mosaic-120.yaml:
```diff
@@ -56,8 +58,16 @@ BandInformation:
-          Notes: DN, 0 - Ok; 1 - no data; 2 - interpolation failed
-Contact:  https://forum.sentinel-hub.com  
+          Notes: DN, 0 - no data; 1 - interpolation passed; 2 - artificially raised value; 3 - interpolation failed
+  QM (Quality Mask) Details: |
```
For now the contents for this field,  won't be displayed, as this new new field is not declared in the details.hbs file .

generally the fields declared in details.hbs are quite general and not collection specific

So I suggest we take these contents to the README.md file under band information
unless it is really neccessary to have it here, then we ll need to declare the field first

mlubej replied:
Yes, I noticed these yaml files are some templates and that this isn't shown. I agree with your suggestion.

Dorothyrono replied:
thanks .. now looks good

mlubej commented:
Thanks @Dorothyrono. `npm run build` works and I checked and removed the problems. Additionally I added some animations and resized them so that they fit the screen.

Dorothyrono commented on collections/sentinel-s2-l2a-mosaic-120.yaml:
```diff
@@ -56,8 +56,8 @@ BandInformation:
           Description: QualityMask
-          Notes: DN, 0 - Ok; 1 - no data; 2 - interpolation failed
-Contact:  https://forum.sentinel-hub.com  
+          Notes: DN, from 0-3; OK if larger than 0. More info under the additional info field.
```
much better to add the link  to the additional info field
here is the link  `https://collections.sentinel-hub.com/sentinel-s2-l2a-mosaic-120/readme.html`

mlubej replied:
I tried and it seems that the link is not rendered, it just shows it how it is. Should it be done in some other way?

![image](https://user-images.githubusercontent.com/37869250/133075601-2766cbbf-8c90-4cae-b30f-a3823bcd85f5.png)

Perhaps I should try with raw html

mlubej replied:
Yes, raw html works. Fixed.

Dorothyrono commented on collections/sentinel-s2-l2a-mosaic-120/README.MD:
```diff
@@ -1,23 +1,46 @@
-## Band information  
-Sentinel-2 L2A 120m mosaic contains 12 Sentinel 2 L2A bands and 1 quality mask. Find more information about sentinel 2 L2A bands [here](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l2a/#available-bands-and-data)
+## Band information
+Sentinel-2 L2A 120m mosaic contains 12 Sentinel 2 L2A bands and 1 quality mask. Find more information about sentinel 2 L2A bands [here](https://docs.sentinel-hub.com/api/latest/data/sentinel-1-l2a/#available-bands-and-data).
```
no idea how we now got a wrong url name  `sentinel -1-l2a `

revert back to sentinel -2-l2a

mlubej replied:
Good catch, no idea how this happened.

mlubej replied:
Fixed.
